030-Data-types/030-data-model.mdx

```diff
@@ -7,6 +7,33 @@ slug: concepts/data-model
 ---
 
+Xata builds on top of Postgres and thus supports the [default data types](https://www.postgresql.org/docs/current/datatype.html) inherit in that platform. Xata additionally adds to that list with unique types such as `file`, `filearray` and `vector` to handle platform features like [file attachements](/docs/sdk/file-attachments) and [vector searching](/docs/sdk/vector-search). In addition, there are some added types to simplify validation of common `text` patterns such as `email` and `multiple`. Usage of these types is ultimately optional if you wish to use Xata only as a Postgres host. If you do decide to utilize them (which is the default way Xata operates through our UI), it is required that you interact with these types through the Xata SDKs.
+
+## Xata specific data types
+
+Below is a mapping of the additional data types Xata provides alongside Postgres types.
+
+| Xata data type          | PostgreSQL data type | Notes                                                 |
+| ----------------------- | -------------------- | ----------------------------------------------------- |
+| [string](#string)       | text                 |                                                       |
+| [text](#text)           | text                 | With constraints on maximum length of 1000 characters |
+| [int](#int)             | bigint               |                                                       |
+| [float](#float)         | double precision     |                                                       |
+| [datetime](#datetime)   | timestamptz          |                                                       |
+| [bool](#bool)           | boolean              |                                                       |
+| [email](#email)         | text                 | Validated by Xata                                     |
+| [multiple](#multiple)   | text[]               | Validated by Xata                                     |
+| [file](#file)           | jsonb                | Validated by Xata                                     |
+| [filearray](#filearray) | jsonb                | Validated by Xata                                     |
+| [link](#link)           | foreign key          |                                                       |
+| [vector](#vector)       | real[]               |                                                       |
+
+## Limitations when using native Postgres types
+
+As you are able to create your own types or use types from extensions in a [dedicated cluster](/docs/dedicated-cluster) through the [Postgres wire protocol](/docs/postgres), there is a possibility that the Xata UI is not aware of the type. For such cases, the UI will display the column values as a string and the corresponding edit UI set to read-only. For now, manipulation of these columns needs to be performed through the wire protocol. Reach out to our [support team](/docs/resources/support) to request adding the data type to Xata.
```

Comment:
For:

> Reach out to our [support team](/docs/resources/support) to request adding the data type to Xata.

This sounded a bit unclear. Perhaps: ... to request adding support for a particular data type to Xata?

Reply:
I think we should just remove this line completely. I'm not sure support will be able to do anything.